app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::UsersController < ActionController::API
+  def ban
+    return render status: :precondition_failed, json: I18n.t('users.invalid_cpf') unless CPF.valid?(cpf_param)
+
+    client = Client.find_by(cpf: cpf_param)
+    personal = Personal.find_by(cpf: cpf_param)
+    return render status: :not_found, json: I18n.t('users.no_account') if client.nil? && personal.nil?
+
+    messages = []
+    user_ban(client, messages)
+    user_ban(personal, messages)
```

Comment:
```suggestion
    user_ban(client || personal, messages)
```

Acho que vocês conseguem usar de curto circuito pra não ter que chamar esse método com algo nil

Reply:
tentei aqui, mas quando mudo pra essa notação só é devolvida uma mensagem :( 

até adicionei dois novos testes para esses casos